src/main/java/uk/ac/cam/cl/dtg/segue/api/UsersFacade.java

```diff
@@ -634,7 +634,9 @@ public Response getUserIdToSchoolMap(@Context final HttpServletRequest httpServl
                                          @QueryParam("user_ids") final String userIdsQueryParam) {
         try {
             RegisteredUserDTO currentUser = userManager.getCurrentRegisteredUser(httpServletRequest);
-            if (!isUserStaff(userManager, currentUser) && !Role.EVENT_LEADER.equals(currentUser.getRole())) {
+            // af599 TODO: Check that including TEACHERs is OK -- old code: !Role.EVENT_LEADER.equals(currentUser.getRole())
+            // af599 TODO: This may not be necessary anymore as we are not doing a school lookup when grabbing event reservations.
+            if (!isUserStaff(userManager, currentUser) && !Arrays.asList(Role.EVENT_LEADER, Role.TEACHER).contains(currentUser.getRole())) {
```

Comment:
I don't think this is a good idea, and if it is, then we'd need to change our Privacy Policy since this is not listed as something we share with teachers presently.

Reply:
Fixed in 2fd148fc.